lbry/lbry/extras/cli.py

```diff
@@ -242,11 +242,12 @@ def run_daemon(args: list, conf: Config):
 
     loop = asyncio.get_event_loop()
 
-    if args.verbose:
-        log.setLevel(logging.DEBUG)
+    modules = args.verbose
+    if modules is not None:
         loop.set_debug(True)
-    else:
-        log.setLevel(logging.INFO)
+        log.setLevel(logging.DEBUG if modules == [] else logging.INFO)
+        for module in modules:
+            logging.getLogger(module).setLevel(logging.DEBUG)
```

Comment:
i think it's still convoluted, also adding more variables usually makes things more complicated (and if you need to add comments then it's even more complicated :-p)

what about (no additional variables and no inline `if` statement):
```
    if args.verbose is not None:
        loop.set_debug(True)
        if len(args.verbose) > 0:
            log.setLevel(logging.INFO)
            for module in args.verbose:
                logging.getLogger(module).setLevel(logging.DEBUG)
        else:
            log.setLevel(logging.DEBUG)
```

Reply:
I agree - it's that I didn't know why it was not readable and thought naming things would help. This could work.